### ISSUE TITLE
Ensure that alerts that appear at the top of the page are announced to the screen reader user.

### ISSUE THREAD

**Issue by ekate** [inveniosoftware/invenio-app-rdm#1191]
in A11y review: "When creating an account or verifying an email address, alerts that appear at the top of the page are not announced to the screen reader user. They will need to navigate through the page until they come upon the information. "

I believe the solution is adding role="alert" to all the "flashed" message, for example:
`<div class="ui info flashed message" role="alert">`
